lib/msteams_hermes/components/adaptive_card.rb

```diff
@@ -14,7 +14,12 @@ module Components
     class AdaptiveCard < Base
       attr_reader :body, :actions, :entities, :width
 
-      def initialize(body:, actions: nil, entities: [], full_width: false)
+      WIDTH = {
```

Comment:
does this work? To incorporate a "private" structure into this class such that any outside caller would still be able to 
`AdaptiveCard.new(body: xyz, with: default)`?

Reply:
I'd encourage you to use a module to encapsulate said constants, instead of a hash.

https://stackoverflow.com/questions/75759/how-to-implement-enums-in-ruby